spec/index.html

```diff
@@ -849,6 +856,12 @@ <h3>Initial Text Direction</h3>
          HTML's `dir` attribute) it can then be correctly presented as:
        </p>
        <div lang="he" dir="rtl">פעילות הבינאום, W3C</div>
+
+       <p>In the absence of an explicit initial text direction, the [[?UAX9]] algorithm
+         can detect text direction from the initial character.
```

Comment:
UAX9 is the right underlying reference, but won't be clear in this context. Might I suggest linking to I18N-glossary's term [Unicode Bidirectional Algorithm](https://w3c.github.io/i18n-glossary/#dfn-unicode-bidi-algorithm) here instead? (Note that the term is exported, so you can just `[=link it=]` in respec.)

---

This definition isn't quite correct. I'm leery of specs such as RDF importing lots of material about bidi because of the complexity of the topic. I think it might be better overall to point back to I18N's materials on this in [Specdev](https://www.w3.org/TR/international-specs/#sec_dir_background), [String-Meta](https://www.w3.org/TR/string-meta/) and the [use cases article](https://www.w3.org/International/articles/lang-bidi-use-cases/).

Correcting this I might suggest:

```suggestion
       <p>In the absence of an explicit initial text direction,
       the [=Unicode Bidirectional Algorithm=] detects the text direction from the content.
       This depends on the first strongly directional character in the text
       or on the context.
       Users need to provide [=bidi isolation=] whenever text is inserted into a larger document
       in order to avoid [=spillover effects=].
```